### PR TITLE
[SP-5713] - Backport of BISERVER-14290 - Schedules created on the User Console runs twice if the user scheduling does not have "Create Content" permission. (9.0 Suite)

### PR DESCRIPTION
**Pull request by bcostahitachivantara**
@pentaho-lmartins 
@ssamora 
@ppatricio 
@smmribeiro

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 36m 31s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl extensions,repository
```
---
:ok_hand: All tests passed!
 
Tests run: **1888**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/platform/job/PR-4758/1/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: extensions/src/main/java/org/pentaho/platform/web/http/api/resources/RepositoryFileStreamProvider.java [hunk 1]
```diff
@@ -125,6 +125,7 @@ public OutputStream getOutputStream() throws Exception {
     RepositoryFileOutputStream outputStream =
         new RepositoryFileOutputStream( tempOutputFilePath, autoCreateUniqueFilename, true );
     outputStream.addListener( this );
+    outputStream.forceFlush( false );
```
Comment (smmribeiro) [inline extensions/src/main/java/org/pentaho/platform/web/http/api/resources/RepositoryFileStreamProvider.java:128]: Please update the copyright year of this file.

**Review by smmribeiro — APPROVED** [review]
Consistent with the original PR.

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 35m 30s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl extensions,repository
```
---
:ok_hand: All tests passed!
 
Tests run: **1888**, Failures: **0**, Skipped: **1** &nbsp;&nbsp; [Test Results](http://wingman.pentaho.net/job/projects/job/platform/job/PR-4758/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by pentaho-lmartins — APPROVED** [review]

**Pull request merged**